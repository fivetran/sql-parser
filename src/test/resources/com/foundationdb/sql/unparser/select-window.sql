SELECT "window".a FROM "window"