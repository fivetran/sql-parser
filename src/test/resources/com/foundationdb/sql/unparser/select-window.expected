SELECT "window".a AS a FROM "window"